pkg/operator/apis/aro.openshift.io/v1alpha1/cluster_types.go

```diff
@@ -57,7 +58,8 @@ type ClusterSpec struct {
 
 // FeaturesSpec defines ARO operator feature gates
 type FeaturesSpec struct {
-	PersistentPrometheus bool `json:"persistentPrometheus,omitempty"`
+	PersistentPrometheus  bool `json:"persistentPrometheus,omitempty"`
+	ManageSamplesOperator bool `json:"manageSamplesOperator,omitempty"`
```

Comment:
When this is rebased, we can use this format for the feature flags:
```suggestion
	ReconcileSamplesOperator bool `json:"reconcileSamplesOperator,omitempty"`
```
https://github.com/Azure/ARO-RP/blob/master/pkg/operator/apis/aro.openshift.io/v1alpha1/cluster_types.go#L65-L75